portingdb/load.py

```diff
@@ -234,7 +239,7 @@ def load_from_directories(db, directories):
                 v['type'] = 'bug'
         print(set(v['type'] for v in values))
         bulk_load(db, values, tables.Link.__table__,
-                  key_columns=['collection_package_id', 'url'])
+                  key_columns=['collection_package_id', 'url', 'last_update'])
```

Comment:
`last_update` shouldn't be here. The `collection_package_id`/`url` combination should be unique by itself.